src/measurement.py

```diff
@@ -1,3 +1,14 @@
+class Error:
+    def __init__(self, lower_error: float, upper_error: float):
+        self.lower_error: float = lower_error
+        self.upper_error: float = upper_error
+
+    def exists(self):
+        return self.lower_error != 0.0 or self.upper_error != 0.0
+    
+    def avg_error(self):
+        return (self.lower_error + self.upper_error) / 2.0
+
```

Comment:
To be really nitpicky: my linter tells me to add two empty lines before a new class starts.